indico/modules/events/registration/lists.py

```diff
@@ -54,11 +56,25 @@ def __init__(self, regform):
             'payment_date': {
                 'title': _('Payment date'),
             },
+            'registration_tags_present': {
+                'title': _('Tags present'),
+                'filter_choices': registration_tag_choices
+            },
+            'registration_tags_absent': {
+                'title': _('Tags absent'),
+                'filter_choices': registration_tag_choices,
+                'filter_only': True
+            },
         }
         self.personal_items = ('title', 'first_name', 'last_name', 'email', 'position', 'affiliation', 'address',
                                'phone', 'country')
         self.list_config = self._get_config()
 
+    def _get_registration_tag_choices(self):
+        tags = RegistrationTag.query.with_parent(self.event).all()
+        tags.sort(key=lambda tag: str.casefold(tag.name))
```

Comment:
In all the places where you use casefold, I'd rather use our `natural_sort_key` util. It also does a case-insensitive sort but it will work nicely when people end up having a digit at the beginning of the tag.